ambry-clustermap/src/main/java/com/github/ambry/clustermap/HelixBootstrapUpgradeUtil.java

```diff
@@ -2306,6 +2326,7 @@ static class ClusterConfigFields {
     private int partitionDiskWeightInGB;
     private int maxPartitionInTransition;
     private int delayRebalanceTimeInMs;
+    private int errorOrRecoverPartitionThresholdForLoadBalance;
```

Comment:
Nit: I added a default configuration file for reference at `config/WagedRebalancerHelixConfig.json`. May be we can add this new config field there too?

Reply:
done